src/ui/public/vislib/__tests__/lib/layout/layout.js

```diff
@@ -52,22 +56,26 @@ dateHistogramArray.forEach(function (data, i) {
     describe('createLayout Method', function () {
       it('should append all the divs', function () {
         expect($(vis.el).find('.vis-wrapper').length).to.be(1);
-        expect($(vis.el).find('.y-axis-col-wrapper').length).to.be(1);
+        expect($(vis.el).find('.y-axis-col-wrapper').length).to.be(2);
         expect($(vis.el).find('.vis-col-wrapper').length).to.be(1);
-        expect($(vis.el).find('.y-axis-col').length).to.be(1);
-        expect($(vis.el).find('.y-axis-title').length).to.be(1);
-        expect($(vis.el).find('.y-axis-div-wrapper').length).to.be(1);
-        expect($(vis.el).find('.y-axis-spacer-block').length).to.be(1);
+        expect($(vis.el).find('.y-axis-col').length).to.be(2);
+        expect($(vis.el).find('.y-axis-title').length).to.be(2);
+        expect($(vis.el).find('.y-axis-div-wrapper').length).to.be(2);
+        expect($(vis.el).find('.y-axis-spacer-block').length).to.be(4);
         expect($(vis.el).find('.chart-wrapper').length).to.be(numberOfCharts);
-        expect($(vis.el).find('.x-axis-wrapper').length).to.be(1);
-        expect($(vis.el).find('.x-axis-div-wrapper').length).to.be(1);
-        expect($(vis.el).find('.x-axis-title').length).to.be(1);
+        expect($(vis.el).find('.x-axis-wrapper').length).to.be(2);
+        expect($(vis.el).find('.x-axis-div-wrapper').length).to.be(2);
+        expect($(vis.el).find('.x-axis-title').length).to.be(2);
```

Comment:
Where are these numbers coming from? Is there any way to make this test more semantic and robust so that we the reasoning in these tests is clearer?